packages/infrastructure/variables.tf

```diff
@@ -0,0 +1,39 @@
+variable "location" {
+  type        = string
+  description = "Primary Location"
+  default     = "westeurope"
+}
+
+variable "name" {
+  type        = string
+  description = "name"
+  default     = "alvtime"
+}
+
+variable "env" {
+  type        = string
+  description = "environment"
+}
+
+variable "aks_service_principal_client_id" {
+  type    = string
+  default = "0.0.0.0"
+}
+
+variable "aks_service_principal_client_secret" {
+  type      = string
+  sensitive = true
+}
+
+variable "sql_server_administrator_login" {
+  type = string
```

Comment:
Do we need to set an sql admin username? Could we not leave it as something default like `root` or something?

Reply:
Might not be needed. You are welcome to do the research.